2024_scraper_national_test.py

```diff
@@ -0,0 +1,223 @@
+import requests
+import json
+import re
+from bs4 import BeautifulSoup #data scraper package
+import numpy
+import pandas as pd
+from itertools import zip_longest
+#url loading and get
+url = 'https://www.realclearpolling.com/polls/president/general/2024/trump-vs-biden#polls'
+page = requests.get(url)
+
+#check and return code
+if page.status_code == 200:
+    url_content = page.text
+    print("Success")
+else:
+    print("Failed to retrieve. Status Code was", page.status_code)
+    exit()
+
+soup = BeautifulSoup(page.content, "html.parser") #EVERYTHING
+
+script_tags = soup.find_all('script')
+
+counter = 0 #this checks the line with the JS that has ALL THE DATA in it
+for script in script_tags:
+    if script.string and 'finalData' in script.string:
+        counter += 1
+        #print(script.string)
+        str = script.string
+        break
```

Comment:
check to make sure that the first tag is the right one for another poll. its just grabbing the first one and breaking

Reply:
i will check bossman
![pepe-listening-to-music](https://github.com/btparrish/election2024-highd-modeling/assets/156129036/b1407251-7bd9-4d15-b59f-8ed8c7dc32b8)